src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -574,9 +574,10 @@ public void AddSourceFile(string fullPath, SourceCodeKind sourceCodeKind = Sourc
             string fullPath,
             SourceCodeKind sourceCodeKind = SourceCodeKind.Regular,
             ImmutableArray<string> folders = default,
+            bool designTimeOnly = default,
```

Comment:
waht does designTimeOnly mean in this context? doc?

Reply:
I'm not sure what to add here.. designTimeOnly doesn't mean anything here which is why its `default`.

Reply:
Agreed, let's doc this. Would anybody ever pass false though?

Reply:
Presumably not, this only has one caller, and it passes in true :)